Sources/Routes/Image.swift

```diff
@@ -34,9 +34,13 @@ public class ImageGIF: Route {
 
     private func packResponseHeaders(body: String, request: Request) -> [String: String] {
         var headersData: [String: String] = [:]
-        headersData["Content-Type"] = "image/gif"
+        headersData["Content-Type"] = getContentType(request: request)
         headersData["Allow"] = "GET" 
         return headersData
     }
+    
+    private func getContentType(request: Request) -> String {
+        return request.path.replacingOccurrences(of: "/", with: "").replacingOccurrences(of: ".", with: "/")
+    }
```

Comment:
I was expecting you to read the value from the `Accept` header, but I guess that can be multiple content types. Just curious, though: Is the `Accept` header present on the requests sent by CobSpec?

Reply:
No, an `Accept` header is not present.